chef_master/source/resource_yum.rst

```diff
@@ -472,9 +472,9 @@ or:
 
 .. tag resource_package_handle_cookbook_file_and_package
 
-.. To handle cookbook_file and yum_package when both called in the same recipe
```

Comment:
This is going to require some more shuffling around, a lot of these tagged regions are referring to yum_package instead of package.